From 997181327d4c55b9d5777583fd630954c45900e3 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Wed, 29 Aug 2018 15:17:35 +0530
Subject: [PATCH] meson: Use same library versioning on macOS as gettext

Requires Meson 0.48, but that hasn't been released yet. It will just be
ignored on older versions of Meson.
---
 meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/meson.build b/meson.build
index 4833043..be5e6f4 100644
--- a/meson.build
+++ b/meson.build
@@ -10,6 +10,8 @@ install_headers('libintl.h')
 intl_shr = shared_library('intl',
   'libintl.c',
   c_args: ['-DSTUB_ONLY'],
+  soversion : 8,
+  darwin_versions : ['10', '10.5'],
   install : true)
 
 intl_sta = static_library ('intl',
-- 
2.17.1

